### ISSUE TITLE
Copy will fail when the copy behavior bind on the touchstart event in iOS Safari

### ISSUE THREAD

**Issue by librory** [lgarron/clipboard-polyfill#114]

**Comment by lgarron** [comment]
Could this be because you have no text selected when the touch starts?
https://github.com/lgarron/clipboard-polyfill/issues/111

**Comment by librory** [comment]
No, I was use it like `clipboard.writeText("hello world");` . But when I attach it on the touchstart event, it will fail. Change the bound event, such as click or touchend, it will be OK. I try to exclude the other influence, so I excute `document.getElementById("plain-copy").addEventListener("touchstart",()=>clipboard.writeText('test'))` in https://garron.net/code/clipboard-polyfill/ . When I touch the first button, console will output 'Unhandled Promise Rejection: Copy command failed.' But when I change touchstart to touchend or click , I will successfully copy the 'test'. This happend in ios safari. I'm not sure if it would happend in android browser.
